### DISCUSSION TITLE
cannot unmarshal array into Go value of type graphql.RawParams

### DISCUSSION THREAD

**Discussion by realdongniu** [99designs/gqlgen#1795] [Q&A]
### What happened?

Hello, when i send an array of graphql requests, 
```
--data-raw '[{"operationName":"permission","query":"query permission {\n  permission {\n    permissions\n    baseApplyUrl\n  }\n}\n"}]' \
  --compressed
```
i got those errors:
```
{"errors":[{"message":"json body could not be decoded: json: cannot unmarshal array into Go value of type graphql.RawParams"}],"data":null}%        
```

### What did you expect?

expect no errors

### versions
 - `gqlgen version`? 0.13
 - `go version`? 1.15
 - dep or go modules?

**Comment by islamaliev** [comment]
I've got the same error when sending a request from a react app with this payload:
```
const payload = `{
    courses {
      title
    }
  }`
```
the corresponding go type is: 
```
type Course {
  id: ID!
  title: String!
  duration: Int!
}
```
Sending the same request with Postman works though

**Comment by boredland** [comment]
I am seeing this error, as soon as I turn on [batching in genql](https://genql.vercel.app/docs/usage/batching-queries), so this may be related.
